reports/lib-utils.api.md

```diff
@@ -667,7 +670,7 @@ export type Selector = Partial<{
 }>;
 
 // @public (undocumented)
-export function setActiveWorkspace(workspace: string): void;
+export function setActiveWorkspaceLocalStorage(workspace: string): void;
```

Comment:
FYI We will publish a new version of lib-utils post merging this PR. We will also update the major version to indicate breaking changes.